### DISCUSSION TITLE
Slack login works, but Google Login is not showing

### DISCUSSION THREAD

**Discussion by houtbyte** [outline/outline#1612] [General]
Hi! 😊 
I am using Google GSuite, and entered my Client ID + Secret. But still, Outline only shows Slack Login Options. I removed all Slack Entries in the .env, but still. Is there ANY documentation how to make this very beautiful but bad documented Wiki system working in a GSuite Environment?

**Comment by tommoor** [comment]
It sounds like you already have a team set up using Slack login? Are you trying to add Google login to the same team?

**Comment by houtbyte** [comment]
Hi Tommoor, can you describe this progress for using Google Workspace?

In development you can quickly get an environment running using Docker by following these steps:

Install these dependencies if you don't already have them
Docker for Desktop
Node.js (v12 LTS preferred)
Yarn
Clone this repo
Register a Slack app at https://api.slack.com/apps
Copy the file .env.sample to .env
Fill out the following fields:
SECRET_KEY (follow instructions in the comments at the top of .env)
SLACK_KEY (this is called "Client ID" in Slack admin)
SLACK_SECRET (this is called "Client Secret" in Slack admin)
Configure your Slack app's Oauth & Permissions settings
Add http://localhost:3000/auth/slack.callback as an Oauth redirect URL
Ensure that the bot token scope contains at least users:read
Run make up. This will download dependencies, build and launch a development version of Outline

**Comment by ceuk** [comment]
Same issue -- Is it possible to support both login types simultaneously? If not, is it possible to transition from slack to google?

**Reply by tommoor** [reply]
It's possible but you'll have to edit the database manually, just update the team record in the database to add the google hosted domain eg `mycompany.com` under the `googleId` column.
